app/controllers/idv_controller.rb

```diff
@@ -36,7 +37,11 @@ def sp_over_quota_limit?
 
   def verify_identity
     analytics.idv_intro_visit
-    redirect_to idv_doc_auth_url
+    if va_inherited_proofing?
```

Comment:
I don't have a strong preference if it lives here vs. `DocAuthController`, but worth pointing out that there are similar redirects that happen there for GPO (verify by mail) verification and in-person proofing enrollment.

https://github.com/18F/identity-idp/blob/4a7fb7c09c126e817ccb9451c37deaad70abe9c1/app/controllers/idv/doc_auth_controller.rb#L4-L5

It does raise a question of whether we'd prefer that the user be _forced_ into the inherited proofing flow if they attempted to navigate directly to the traditional unsupervised proofing flow (i.e. navigated to `/verify/doc_auth`).

Reply:
Good point, that there are many redirect conditions in the OpenidConnect::Auth, Idv, and DocAuth controllers. It seems odd that we would redirect to inherited proofing (IP) in DocAuth, esp. since we know IP will be required of the user when they first hit the Openid controller (unless, there is required logic in DocAuth?). Does it make sense that DocAuth is a default, or that the user is somehow directed to /verify/doc_auth (apart from Idv)?

Reply:
@aduth If we _do not_ force them into the IP flow, that may very well mess with the analytics/metrics we want to capture regarding whether or not the user is able to traverse the whole IP process successfully. However, what you bring up may also provide equally valuable analytics/metrics if the user _does_ in fact decide (for whatever reason) to abandon the IP flow and (for example) go with the unsupervised flow - was it because they got lost, did they become frustrated, were they unable to complete due to errors, etc.? Good question to raise.

Reply:
@jscodefix Everything comes through this controller, but since this was previously redirecting everything to DocAuthController, it was a de facto default entrypoint for identity verification. I'm not familiar enough with inherited proofing to know if it's different enough to make it awkward to try to include there, vs. separately (as you've done here).

I think this could be fine, if:

- We think inherited proofing is different enough from the traditional identity verification flow
- We're okay with the `idv_intro_visit` analytics being logged
- We're okay that someone could still find their way into the DocAuth flow
   - I wouldn't think this is common, but someone could manually edit the URL, or visit from a different partner requiring identity verification.

Reply:
@aduth Responding to your 3 points above:
1. To me, IP verification is significantly different than DocAuth; their PII is pulled from the SP via an API call.
2. I don't know the intent of the (idv_intro_visit) analytics, but still seems applicable here. I'll add the question to Jira and discuss with the team.
3. If an IP user doesn't arrive through the OIDC controller, then IP will not work.